lib/order.rb

```diff
@@ -0,0 +1,40 @@
+require_relative 'send_sms'
+require_relative 'dishes'
+
+class Order
+
+  attr_reader :selection, :dishes, :sms
+
+  def initialize(dishes = Dishes.new, sms = SendSMS.new)
+    @dishes = dishes
+    @sms = sms
+  end
+
+  def view_dishes
+    dishes.view
+  end
+
+  def create(selection)
+    @selection = selection
+  end
+
```

Comment:
Could there have been and additional "review_basket" method in here? Or something similar. So that the user can review their order before verifying?